### DISCUSSION TITLE
Chataigne sample files

### DISCUSSION THREAD

**Discussion by 7sorkr** [ponoor/step-series-support#21] [Show and tell]
It is a program that is frequently used because of its small capacity and intuitiveness.
Text is only supported in English.

![STEP400_OSC](https://user-images.githubusercontent.com/32606306/183347633-e68cf3a2-c9ca-4ad5-8bf1-915e2b41d8f6.PNG)

**Download** https://benjamin.kuperberg.fr/chataigne/en#download
**github** https://github.com/benkuper/Chataigne
**Discord** https://discord.gg/Z8KzxTzt6K

Sample File
[STEP_400_OSC_0808.zip](https://github.com/ponoor/step-series-support/files/9279000/STEP_400_OSC_0808.zip)

The more you touch, the more amazing STEP 400
always have a nice day

**Comment by kanta** [comment]
@7sorkr I haven't heard about Chataigne, but this looks very useful for many kind of people, and your project should be perfect one for STEP series! Thank you for sharing this project and let me try! And yes, of course I will add this to the example files.
Thank you for your contribution!!

**Reply by 7sorkr** [reply]
I just created an example file.
We will update you as we learn more about the feature.

thank you
